tiledb/api/c_api/config/config_api_external.h

```diff
@@ -458,6 +458,9 @@ TILEDB_EXPORT void tiledb_config_free(tiledb_config_t** config) TILEDB_NOEXCEPT;
  *    The scale factor for exponential backoff when connecting to S3.
  *    Any `long` value is acceptable. <br>
  *    **Default**: 25
+ * - `vfs.s3.curl_header` <br>
+ *    Header to add to curl requests with format "header_key header_value" <br>
```

Comment:
On Windows the AWS SDK uses the WinHTTP API instead of curl. The option should have a name that does not mention curl, like `vfs.s3.custom_headers`.